src/components/SideBar.tsx

```diff
@@ -30,6 +40,7 @@ const CourseItem = styled.li`
 		list-style: none;
 	}
 `;
+
```

Comment:
```
${({ selected }) => selected ? css`myNewProperty: someValue` : ''}
````

Reply:
This was a cool trick to learn and implement!